### PR TITLE
Added Rich Presence for SpinSha.re

### PR DESCRIPTION
**Pull request by jy1263**
---
SpinShare
---

About: [SpinSha.re](https://github.com/SpinShare/) is a custom chart sharing service for the rhythm game, Spin Rhythm XD.

---
Screenshots
---
- Idling:
image

- Viewing Song: 
image

- Viewing Profile:
image

- Viewing Frontpage: 
image

### CODE REVIEW

File: websites/S/Spinsha.re/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    "service": "SpinShare",
+    "description": {
+      "en": "SpinShare is a chart sharing service for Spin Rhythm XD."
+    },
+    "url": "spinsha.re",
+    "regExp": "(spinsha[.]re)[/]",
+    "version": "1.1.0",
+    "logo": "https://spinshare.b-cdn.net/assets/img/favicon.png",
+    "thumbnail": "https://spinshare.b-cdn.net/assets/img/clientScreenshots/newsongs.png",
```
Comment (Alanexei) [inline websites/S/Spinsha.re/dist/metadata.json:15]: Upload these to imgur.

File: websites/S/Spinsha.re/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "author": {
+      "name": "ayanamy.jy126",
+      "id": "243279723487035393"
+    },
+    "service": "SpinShare",
+    "description": {
+      "en": "SpinShare is a chart sharing service for Spin Rhythm XD."
+    },
+    "url": "spinsha.re",
+    "regExp": "(spinsha[.]re)[/]",
```
Comment (Alanexei) [inline websites/S/Spinsha.re/dist/metadata.json:12]: Is there a good reason for this? Otherwise you could just get away with just the URL.

**Review by Alanexei — CHANGES_REQUESTED** [review]

File: websites/S/Spinsha.re/dist/metadata.json (hunk 2 above)
Comment (jy1263) [reply websites/S/Spinsha.re/dist/metadata.json:12]: o okay thanks

File: websites/S/Spinsha.re/dist/metadata.json (hunk 1 above)
Comment (jy1263) [reply websites/S/Spinsha.re/dist/metadata.json]: okay thx, i've changed the links

File: websites/S/Spinsha.re/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      "id": "243279723487035393"
+    },
+    "service": "SpinShare",
+    "description": {
+      "en": "SpinShare is a chart sharing service for Spin Rhythm XD."
+    },
+    "url": "spinsha.re",
+    "version": "1.1.0",
+    "logo": "https://i.imgur.com/7DKJ3DZ.png",
+    "thumbnail": "https://i.imgur.com/oWVD4Pr.png",
```
Comment (Bas950) [inline websites/S/Spinsha.re/dist/metadata.json:14]: logo must be 512x512
and the thumbnail must be a promotional banner of the website or a screenshot of the website, not a screenshot of what it shows in your Discord.

File: websites/S/Spinsha.re/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "author": {
+      "name": "ayanamy.jy126",
+      "id": "243279723487035393"
+    },
+    "service": "SpinShare",
+    "description": {
+      "en": "SpinShare is a chart sharing service for Spin Rhythm XD."
+    },
+    "url": "spinsha.re",
+    "version": "1.1.0",
```
Comment (Bas950) [inline websites/S/Spinsha.re/dist/metadata.json:12]: version 1.0.0 on first release

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/Spinsha.re/dist/metadata.json (hunk 3 above)
Comment (jy1263) [reply websites/S/Spinsha.re/dist/metadata.json:14]: o okay

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**